src/audio/kpb.c

```diff
@@ -159,9 +157,19 @@ static int kpb_prepare(struct comp_dev *dev)
 {
 	struct comp_data *cd = comp_get_drvdata(dev);
 	int ret = 0;
-	int i;
+	int i = 0;
 	struct list_item *blist;
 	struct comp_buffer *sink;
+	struct hb *history_buffer = &cd->history_buffer;
+	struct hb *new_hb = NULL;
+	/*! total allocation size */
+	size_t hb_size = KPB_MAX_BUFFER_SIZE;
+	/*! current allocation size */
+	size_t ca_size = hb_size;
+	/*! memory caps priorites for history buffer */
+	int hb_mcp[KPB_NO_OF_MEM_POOLS] = {SOF_MEM_CAPS_LP, SOF_MEM_CAPS_HP,
+				       SOF_MEM_CAPS_RAM};
+	void *ptr = NULL;
```

Comment:
@mrajwa ptr is a very generic name. can we have a better name to indicate what it is for?

Reply:
Changed.